product_listings_manager/models.py

```diff
@@ -0,0 +1,131 @@
+"""Models definition relecting tables in composedb.
+
+These models are not fully relecting the composedb schema:
+
+- Tables and columns which not needed by PLM are not included.
+
+- Length of db.String() column is not required by postgresql(composedb uses it), in case of
+testing with other db backend(e.g. sqlite) a value is given in following definition and it
+has no side effect to postgresql(composedb).
+"""
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+
+class Packages(db.Model):
+    """packages table in composedb.
+
+    Only needed columns in packages table are defined here.
+    """
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255), nullable=False)
+    arch = db.Column(db.String(32), nullable=False)
+    version = db.Column(db.String(255), nullable=False)
+
+    def __repr__(self):
+        return '<Package %d %s %s %s>' % (self.id, self.name, self.version, self.arch)
+
+
+trees = db.Table(
```

Comment:
Is the name significant here? I would be a little worried that it could be confusing that table `tree_product_map` is represented by variable `trees`. Especially given that there is a table named `trees`.

Reply:
I changed the name.